composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": "^5.5|^7.0",
         "symfony/symfony"                : "^2.7|^3.0",
-        "react/react"                    : "^0.4"
+        "react/react"                    : "0.4.1"
```

Comment:
you could even fix it at `0.4.3`

=> https://github.com/reactphp/http/blob/v0.4.3/src/Server.php#L61